### ISSUE TITLE
add comment without author

### ISSUE THREAD

**Issue by d3zn** [qax-os/excelize#1204]
**Description**
author参数设置为空字符串时 ，打开excel提示 “部分内容有问题。是否让我们尽量尝试恢复? ...”
When the author parameter is set to an empty string, opening excel prompts "There is a problem with some content. Let's try to restore it as much as possible? ..."  (google translate)

**code**

```golang
f := excelize.NewFile()
sheet := "Sheet1"
_ = f.SetCellStr(sheet, "A1", "value")
f.AddComment(sheet, "A1", fmt.Sprintf(`{"author":"", "text":"%s"}`, "value comment"))
f.SaveAs("test.xlsx")
```

**Output of `go version`:**

```text
1.16.5
```

**Excelize version or commit ID:**

```text
2.6.0
```
**Environment details (OS, Microsoft Excel™ version, physical, etc.):**
```text
macOS BigSur 11.5.2  Microsoft Excel for Mac version 16.59 (22031300) Microsoft 365
```

**Comment by d3zn** [comment]
when use ```fmt.Sprintf(`{"author":"", "text":"%s"}`, "value comment")```，excel comment has default author: `Author: value comment` , but i dont need

**Comment by xuri** [comment]
Thanks for your issue. The author will show with bold style in the comments box currently. I'll consider adding support for creating rich text comments later.

**Comment by xuri** [comment]
I have added support for creating rich text comments. This feature will be released in version 2.7.0, this is a breaking change, now you can add a comment like this:

```go
err := f.AddComment(sheet, excelize.Comment{
    Cell:   "A12",
    Author: "Excelize",
    Text:   "Excelize: This is a comment.",
})
```

or set rich text in the comment

```go
err := f.AddComment(sheet, excelize.Comment{
    Cell:   "A12",
    Author: "Excelize",
    Runs:   []excelize.RichTextRun{
        {Text: "Excelize: ", Font: &excelize.Font{Bold: true}},
        {Text: "This is a comment."},
    },
})
```
